ivy/functional/frontends/paddle/nn/functional/norm.py

```diff
@@ -8,3 +8,71 @@
 @with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
 def layer_norm(x, normalized_shape, weight=None, bias=None, epsilon=1e-05, name=None):
     return ivy.layer_norm(x, normalized_shape, weight, bias, epsilon)
+
+
+# instance_norm
+@to_ivy_arrays_and_back
+@with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
+def instance_norm(
+    x,
+    running_mean=None,
+    running_var=None,
+    weight=None,
+    bias=None,
+    use_input_stats=True,
+    momentum=0.9,
+    epsilon=1e-05,
+    data_format="NCHW",
+    name=None,
+):
+    r"""
```

Comment:
There is no need for docstrings in the frontend API, can you please remove them. Thanks!